cmd/snap/main.go

```diff
@@ -508,6 +530,14 @@ var wrongDashes = string([]rune{
 	0x2e3b, // three-em dash
 })
 
+type unknownCommandError struct {
+	errStr string
```

Comment:
my preference is for this to be just "msg" (I looked at the stdlib itself and it never seems to use "errStr" but "msg", "errorString", or just rarely "err" even if it's not an error)